release-common.sh

```diff
@@ -17,6 +17,15 @@ print_header () {
     echo -e "\033[1;33m$1\033[0m"
 }
 
+print_header_with_line () {
+    echo -e "\033[1;33m$1\033[0m"
+    echo -en "\033[1;33m"
+    for ((i=0; i<${#1}; i++)); do
+        echo -n "="
+    done
+    echo -e "\033[0m"
+}
```

Comment:
```suggestion
print_header_with_line () {
    print_header "$1"
    print_header "$(tr '[:print:]' = <<< "$1")"
}
```

Reply:
I knew there had to be a better way. Thanks!

Although I'm surprised space is in `[:print:]`

Reply:
> Although I'm surprised space is in `[:print:]`

Maybe you are thinking of `[:graph:]`.

From the `tr` manpage:

```
       [:graph:]
              all printable characters, not including space

       [:print:]
              all printable characters, including space
```